sensor:
  sensor_model_p_occ: 0.75
  sensor_model_p_free: 0.45
  sensor_model_p_prior: 0.5

frame:
  robot_frame: base_link
  map_frame: map

mapping:
  map_size_x: 50.0
  map_size_y: 50.0
  map_resolution: 0.05
  map_publish_freq: 1.0
  update_movement: 0.1

file_name: gridmap # save
